html-tables/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head><title>Html Tables</title></head>
+  <body>
+    <h1>Student Grade Table</h1>
+
+    <table>
+    <tr>
+  <th scope="col">Student Name</th>
+  <th scope="col">Course</th>
+  <th scope="col">Grade</th>
```

Comment:
Woah, this indentation got messed up. I'm honestly not sure how you are committing this, you should be getting warnings when you try to commit 🤔